### ISSUE TITLE
Use trained model on new, unseen and unannotated images

### ISSUE THREAD

**Issue by Selina9** [matterport/Mask_RCNN#1335]
Hi,

So I have used the documentation to train my own model and tested it on my validation set of images.
However, is this just not using the supplied annotated masks I made, from the validation image set?

I want to show the model a new image, that has not been annotated to see how well it performs in segmenting it.

How do I go about doing that?

Thanks in advance!

P.S, how do I get the tensorboard training graphs to monitor my loss?

**Comment by freezurbern** [comment]
What you want is called 'inference'. I have found the following tutorial useful:
http://www.immersivelimit.com/tutorials/using-mask-r-cnn-on-custom-coco-like-dataset
The code from the tutorial for inferencing can be found here:
https://github.com/akTwelve/tutorials/blob/master/mask_rcnn/MaskRCNN_TrainAndInference.ipynb

As for TensorBoard, I use this Mask RCNN in a Docker container. There, I use the following compose file:
```
version: "3.7"
services:
  tensorboard:
    image: tensorflow/tensorflow:1.13.1-gpu-py3-jupyter
    container_name: tensorboard
    restart: always
    init: true
    ports: 
      - "6006:6006"
    volumes:
    - ./notebooks:/notebooks
    command: >
      bash -c "source /etc/bash.bashrc && tensorboard --logdir="/notebooks" --port 6006"
```

You may adapt the ```tensorboard --logdir="/notebooks" --port 6006``` portion to your environment.

**Comment by Selina9** [comment]
Hi,

Thank you for your response!
I've managed to fix what I needed with the help of that code you linked :)

I'm not using a docker container. Is there another way to utilize tensorboard?

**Comment by josephofiowa** [comment]
@Selina9 Yup! https://www.tensorflow.org/tensorboard/get_started The [notebook linked there](https://colab.research.google.com/github/tensorflow/tensorboard/blob/master/docs/get_started.ipynb) is quite useful.
